lib/chrono_model/conversions.rb

```diff
@@ -15,7 +15,7 @@ def string_to_utc_time(string)
     end
 
     def time_to_utc_string(time)
-      [time.to_s(:db), sprintf('%06d', time.usec)].join '.'
+      [time.to_formatted_s(:db), sprintf('%06d', time.usec)].join '.'
```

Comment:
Please split this change in a different commit, it will be useful for the rebase

I would like to reduce the number of changes and commits in this branch before merging